ide/css.editor/src/org/netbeans/modules/css/editor/module/main/properties/text.properties

```diff
@@ -101,6 +101,8 @@ overflow-wrap=normal | break-word | <var-fn>
         ]\
          | <var-fn>
 
+user-select=auto | none | text | all
```

Comment:
1. Please add ` | <var-fn>`, so that a variable based setting is supported
2. I have mixed feelings about `contain`. The whole CSS4 standard is a draft, I see that the reality is already here, but why not then use the draft standard as basis? Users will anyway have to decide how much they trust into the CSS enging.